lib/cli/consolecommand.cpp

```diff
@@ -382,10 +385,17 @@ int ConsoleCommand::RunScriptConsole(ScriptFrame& scriptFrame, const String& add
 			line = commandOnce;
 
 		if (!line.empty() && line[0] == '$') {
-			if (line == "$continue")
+			if (line == "$continue" || line == "$quit" || line == "$exit")
 				break;
+			else if (line == "$help")
+				std::cout << "Welcome to the Icinga 2 console/script debugger.\n"
+					"Usable commands:\n"
+					"  $continue, $quit, $exit   Quit the console\n"
```

Comment:
While that's technically correct (the best kind of correct...) this might be confusing for users: "Why do we have $continue for closing the console?"